packages/experimental/Icon/src/Icon.tsx

```diff
@@ -51,29 +52,25 @@ function renderFontIcon(iconProps: IconProps) {
 
 function renderSvg(iconProps: IconProps) {
   const svgIconProps: SvgIconProps = iconProps.svgSource;
-  const { width, height } = iconProps;
+  const { width, height, color } = iconProps;
   const viewBox = iconProps.svgSource.viewBox;
   const style = mergeStyles(iconProps.style, rasterImageStyleCache({ width: width, height: height }, [width, height])[0]);
 
   // react-native-svg is still on 0.61, and their color prop doesn't handle ColorValue
   // If a color for the icon is not supplied, fall back to white or black depending on appearance
   const theme = useFluentTheme();
-  const iconColor = svgIconProps.color
-    ? svgIconProps.color
-    : getCurrentAppearance(theme.host.appearance, 'light') === 'dark'
-    ? '#FFFFFF'
-    : '#000000';
+  const iconColor = color ? color : getCurrentAppearance(theme.host.appearance, 'light') === 'dark' ? '#FFFFFF' : '#000000';
 
   if (svgIconProps.src) {
     return (
       <View style={style}>
-        <svgIconProps.src viewBox={viewBox} width={width} height={height} color={iconColor} />
+        <svgIconProps.src viewBox={viewBox} width={width} height={height} color={iconColor as Color} />
```

Comment:
can we add a comment that links out to the github.com/flurn issue over ColorValue/PlatformColor support dependency from rn-svg, and link this line to the issue as a workaround to remove [can do after commit]?

Reply:
Do you have a link on this issue?

Reply:
https://github.com/react-native-svg/react-native-svg/issues/1391 I think is the right one to reference. The tl;dr is RN-SVG was never updated for RN 0.63 to support the ColorValue type, which lets you do platform specific colors on top of your standard RBGA colors. I've seen PR's that try to implement the support, but only for iOS/Android, and not in a generic way where things like react-native-macos/windows work out of the box. It's also extra complex because you can't necessarily convert a ColorValue type to a RGB color: it could represent a system color that changes with light/dark/high contrast, so one hex value isn't sufficient. 

I'm not sure how RN-win32 handles colors with their internal implementation of RN-SVG, you might already have support for ColorValue for all I know. I think it's just a question of funding / prioritization for closing that issue in the community module.

Reply:
@Saadnajmi I believe [without re-checking] rn-win32 handles ColorValue/PlatformColors in the same folly object to color info flow that SVG color props are handled by, so getting the ColorValue is already supported IIRC

Reply:
https://github.com/microsoft/fluentui-react-native/issues/728 we should reference our own issue if we add it to a comment IMO